base_layer/p2p/src/initialization.rs

```diff
@@ -97,15 +97,12 @@ impl CommsInitializationError {
     pub fn to_friendly_string(&self) -> String {
         // Add any helpful user-facing messages here
         match self {
-            CommsInitializationError::HiddenServiceBuilderError(
-                tor::HiddenServiceBuilderError::HiddenServiceControllerError(
-                    tor::HiddenServiceControllerError::TorControlPortOffline,
-                ),
-            ) => r#"Unable to connect to the Tor control port.
-Please check that you have the Tor proxy running and that access to the Tor control port is turned on.
-If you are unsure of what to do, use the following command to start the Tor proxy:
-tor --allow-missing-torrc --ignore-missing-torrc --clientonly 1 --socksport 9050 --controlport 127.0.0.1:9051 --log "notice stdout" --clientuseipv6 1"#
-                .to_string(),
+            CommsInitializationError::HiddenServiceControllerError(HiddenServiceControllerError::TorControlPortOffline)
```

Comment:
```rust
CommsInitializationError::HiddenServiceBuilderError(
                tor::HiddenServiceBuilderError::HiddenServiceControllerError(
                    tor::HiddenServiceControllerError::TorControlPortOffline,
                ),
            ) 
```

Can this error still happen? We may be undoing another case - ideally, make sure that the above cannot happen (refactor as necessary), or handle both cases (quick n dirty)

```rust
CommsInitializationError::HiddenServiceBuilderError(
                tor::HiddenServiceBuilderError::HiddenServiceControllerError(
                    tor::HiddenServiceControllerError::TorControlPortOffline,
                ),
            ) |  CommsInitializationError::HiddenServiceControllerError(HiddenServiceControllerError::TorControlPortOffline)  => 
```

Reply:
In all my testing I could not hit the other error. I don't know if that error can still happen, but it's definitely not going to be a tor not connected issue. If tor is disconnected, it only hits the current on in this PR.

Reply:
Remove HiddenServiceControllerError from comms/src/tor/hidden_service/builder.rs and see if it compiles